internal/services/machinelearning/machine_learning_compute_cluster_resource.go

```diff
@@ -180,6 +188,7 @@ func resourceComputeClusterCreate(d *pluginsdk.ResourceData, meta interface{}) e
 		VMPriority:             &vmPriority,
 		ScaleSettings:          expandScaleSettings(d.Get("scale_settings").([]interface{})),
 		UserAccountCredentials: expandUserAccountCredentials(d.Get("ssh").([]interface{})),
+		EnableNodePublicIP:     utils.Bool(!d.Get("local_auth_enabled").(bool)),
```

Comment:
you can use the new pointer functions here
```suggestion
		EnableNodePublicIP:     pointer.To(!d.Get("local_auth_enabled").(bool)),
```

Reply:
Done.